btrack/utils.py

```diff
@@ -109,201 +73,25 @@ def log_stats(stats):
     )
 
 
-def read_motion_model(config: dict) -> MotionModel:
-    """Read a motion model from a configuration dictionary.
-
-    Read in a motion model description file and return a dictionary containing
-    the appropriate parameters. See `models.MotionModel` for more details of the
-    parameters.
-
-    Parameters
-    ----------
-    config : dict
-        A dictionary describing the motion model.
-
-    Returns
-    -------
-    model : MotionModel
-        A `models.MotionModel` instance to configure BayesianTracker.
-
-    Notes
-    -----
-    Motion models can be described using JSON format, with a basic structure
-    as follows:
-
-        {
-          "MotionModel":{
-            "name": "ConstantVelocity",
-            "dt": 1.0,
-            "measurements": 3,
-            "states": 6,
-            "accuracy": 2.0,
-            "A": {
-              "matrix": [1,0,0,1,0,0,...
-              ...
-              ] }
-            }
-        }
-
-    Matrices are flattened JSON arrays.
-
-    Most are self explanatory, except accuracy (perhaps a misnoma) - this
-    represents the integration limits when determining the probabilities from
-    the multivariate normal distribution.
-
-    Note that the matrices are stored as 1D matrices here. In the future,
-    this could form part of a Python only motion model.
-
-    TODO(arl): More parsing of the data/reshaping arrays. Raise an
-    appropriate error if there is something wrong with the model
-    definition.
-    """
-
-    if "MotionModel" not in list(config.keys()):
-        raise ValueError("Not a valid motion model in configuration.")
-
-    motion_config = config["MotionModel"]
-    if not motion_config:
+def read_motion_model(cfg: dict) -> MotionModel:
+    cfg = cfg.get("MotionModel", None)
+    if not cfg:
```

Comment:
Actually, this is slightly weird since the json file may return an empty dictionary, in which case this works. Perhaps just changing to `.get("MotionModel", {})` would cover both cases

Reply:
That certainly seems like a good option. I get nervous when `not` is used with `None` without a use of `is None`